log/logger.go

```diff
@@ -0,0 +1,88 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/CMSgov/bcda-ssas-app/ssas/constants"
+	"github.com/sirupsen/logrus"
+)
+
+// Logger provides a structured logger for this service
+var Logger logrus.FieldLogger
+
+// Event contains the superset of fields that may be included in Logger statements
+type Event struct {
```

Comment:
We have this `Event` struct and the `init()` for the logger defined in ssas/logger.go - can we remove one of the dupes? 😃